docusaurus/docs/protocol/tokenomics/token_logic_modules.md

```diff
@@ -0,0 +1,446 @@
+for themselves by sending fake traffic. This is shown in red.**
+
+```mermaid
+---
+title: "Self Dealing Attack"
+---
+sequenceDiagram
+    actor U as End User
+    participant AG as Application | Gateway
+    participant S as Supplier
+    participant P as Protocol
+    participant SO as Service Owner
+    participant DAO as DAO
+    participant BP as Block Proposer
+
+    loop "provide service throughout session"
+        U ->> +AG: RPC Request
+        AG ->> +S: POKT Relay Request
+        S ->> -AG: POKT Relay Response
+        AG ->> -U: RPC Response
+    end
+
+    critical "settle session accounting"
+        AG -->> +P: Pay POKT (based on work done)
+        P -->> -S: Receive POKT (based on work done)
+        note over AG,P: Inflation: Mint new POKT
+        rect rgb(247, 13, 26)
+            P -->> S: Send rewards (% of mint)
+            P -->> SO: Send mint rewards (% of mint)
+        end
+        P -->> DAO: Send mint rewards (% of mint)
+        P -->> BP: Send mint rewards (% of mint)
+    end
+```
+
+### Reimbursement Request Philosophy
+
+_Solving the above problem is non-trivial_.
+
+See the [resources](./resources.md) for more information on the long-term game-theoretic solutions.
+
+In the meantime, the interim manual approach described below is a stepping stone
+do things that don't scale in the short term, but can be easily automated, while
+enabling permissionless demand and dissuading self-dealing attacks.
+
+### Reimbursement Request Design
+
+This TLM is a dependency of the Global Mint TLM; i.e., it **MUST** be active ONLY IF Global Mint is active.
+
+This TLM can, **theoretically**, be removed if self-dealing attacks are not a concern,
+or if the global mint per claim governance parameter is set to zero.
+
+The goal of the TLM is supplement the Global Mint TLM such that:
+
+1. The application is overcharged by the inflation amount in `TLM: Global Mint`.
+2. The application must **"show face"** in front of the DAO/PNF to request reimbursement.
+3. PNF needs to **manually approve** the reimbursement request.
+
+**While this is not perfect, it follows on the **[Deterrence Theory](<https://en.wikipedia.org/wiki/Deterrence_(penology)>)** that
+the increased risk of punishment will dissuade bad actors.**
+
+_NOTE: A side effect of this TLM is creating additional buy pressure of the token as Applications
+and Gateways will be responsible for frequently "topping up" their balances and app stakes._
+
+```mermaid
+---
+title: "Token Logic Module: Global Mint Reimbursement Request"
+---
+flowchart TD
+    SA(["Settlement Amount (SA)"])
+    PCI(["Per Claim Global Inflation (PCGI)"])
+    ARRE{{Application Reimbursement <br> Request Event}}
+    TM2["Tokenomics Module"]
+
+    subgraph TLMGM["TLM: Global Mint"]
+        IA(["Inflation Mint Coin (IA)"])
+        ID["Inflation Distribution <br> (see TLM above for details)"]
+        GP["Governance Params"]
```

Comment:
Is there a reason that this node is not rounded like the other governance params?

```suggestion
        GP(["Governance Params"])
```